draft-terbu-sd-jwt-vc.md

```diff
@@ -84,25 +85,22 @@ involved, a Status Provider, who delivers revocation information to Verifiers.
 (The Verifier can also serve as the Status Provider.)
 
 This specification defines Verifiable Credentials based on the SD-JWT
-format with a JWT Claim Set.
+format with a JWT Claim Set. It can be used when there are no selective disclosable claims, too.
 
 ## Rationale
 
 JSON Web Tokens (JWTs) [@!RFC7519] can in principle be used to express
 Verifiable Credentials in a way that is easy to understand and process as it
-builds upon established web primitives. While JWT-based credentials enable selective
-disclosure, i.e., the ability for a Holder to disclose only a subset of the contained
-claims, in an Identity Provider ecosystem by issuing new JWTs to the Verifier for
-every presentation, this approach does not work in the three-party-model.
+builds upon established web primitives.
 
 Selective Disclosure JWT (SD-JWT) [@!I-D.ietf-oauth-selective-disclosure-jwt] is
 a specification that introduces conventions to support selective disclosure for
 JWTs: For an SD-JWT document, a Holder can decide which claims to release (within
-bounds defined by the Issuer). This format is therefore perfectly suited for
-Verifiable Credentials.
+bounds defined by the Issuer).
 
 SD-JWT itself does not define the claims that must be used within the payload or
```

Comment:
suggest to reorganize this paragraph 
Move 2nd and 3rd sentence to previous paragraph forward (all related to sd and non sd). Modify 3rd sentence.

"This specification therefore defines how Verifiable Credentials
can be expressed using SD-JWT covering both options, with and without selective disclosure."

1st sentence fits better with the last paragraph as this is about the content. Suggest to add the following sentence between this sentence and the rest: 
"SD-JWT VC defines a content model based on JWT claims."

Reply:
I agree, when the section above is removed, the paragraphs don't really fit together nicely

Reply:
thanks - I think it reads better now.